<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationDlg</class>
 <widget class="QDialog" name="CalibrationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>相机标定</string>
  </property>
  <widget class="QLabel" name="leftImage_disp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rightImage_disp">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="openCamBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalibraBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>标定参数确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存结果</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="resultTxtEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>681</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>681</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>输入参数：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>列角点数：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="rowCorPointNum_Let"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="colCorPointNum_Let"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>棋盘格宽</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>棋盘格长：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="perBoardwidthEdt"/>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="perBoardheightEdt"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>华文行楷</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>行角点数：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
